guides/installing_esphome.rst

```diff
@@ -49,19 +49,37 @@ with the following:
 Mac
 ---
 
-There are no tested installation instructions for Mac. ESPHome does support
-Mac & will run with no problem.
+ESPHome supports macOS and will run without issues.
```

Comment:
This is a bold promise. Perhaps just say that ESPHome supports macOS, and there are several ways to install it.